frontend/package.json

```diff
@@ -7,13 +7,17 @@
     "bulma": "^0.7.2",
     "bulma-extensions": "^4.0.1",
     "classnames": "^2.2.6",
+    "connected-react-router": "^6.3.0",
     "mediaquery": "0.0.4",
     "node-sass-chokidar": "^1.3.4",
     "npm-run-all": "^4.1.5",
     "prop-types": "^15.6.2",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "react-redux": "^6.0.0",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
+    "react-router-redux": "^4.0.8",
```

Comment:
I'm still amazed by the number of dependencies we have to add in order to make a simple thing working 💥 😲

Reply:
we can supress react-router-redux, but react-router & react-router-dom are needed for a router !

Reply:
alright, good job <3